js/data/bucket/line_bucket.js

```diff
@@ -347,6 +359,12 @@ LineBucket.prototype.addLine = function(vertices, join, cap, miterLimit, roundLi
         startOfLine = false;
     }
 
+    this.populatePaintArrays(
+        'line', {zoom: this.zoom},
+        feature.properties,
```

Comment:
If we only use `feature.properties`, should we pass `featureProperties` in the outer function directly?